packages/core/src/components/graphs/line.ts

```diff
@@ -44,7 +44,14 @@ export class Line extends Component {
 		this.lineGenerator = line()
 			.x((d, i) => this.services.cartesianScales.getDomainValue(d, i))
 			.y((d, i) => this.services.cartesianScales.getRangeValue(d, i))
-			.curve(this.services.curves.getD3Curve());
+			.curve(this.services.curves.getD3Curve())
+      .defined((d: any, i) => {
+				if (!d || d.value === null) {
```

Comment:
Might be useful to include the check to `undefined` as well.

I'm not sure about the `!d` check:
 - if `d` is always an object, it will always pass
 - if `d` can be a number, then this check is improvable because it will be incorrectly matched by 0
 - I don't believe `d` can be null or undefined

Thanks for any clarification :)

Reply:
👍